docs/guide/tutorial-6-4-wheels-robot.md

```diff
@@ -159,10 +164,10 @@ As already says in [Tutorial 3](tutorial-3-appearance.md), the distance sensor r
 %end
 
 > **Hands on #5**: Add the two distance sensors as explained above.
-The distance sensors are at an angle to 0.3 [rad] with the robot front vector.
-Set their graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
-Set their color to blue.
-Set their `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
+The distance sensors are at an angle to 0.3 [rad] with the front vector of the robot.
+Set thei graphical and physical shape to a cube (not transformed) having a edge of `0.01` [m].
+Set thei color to blue.
+Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-devices).
```

Comment:
```suggestion
Set thei `name` field according to the labels of [this figure](#top-view-of-the-4-wheels-robot-the-grid-behind-the-robot-has-a-dimension-of-0-2-x-0-3-meters-the-text-labels-correspond-to-the-name-of-the-device).
```

Reply:
Oh yes, I see my mistake here. Thanks a lot for pointing them out :)

Reply:
You're welcome, thank you for your help making the doc better.